docroot/modules/custom/mayflower/src/Helper.php

```diff
@@ -1762,10 +1762,22 @@ public static function prepareViewsHeader(ViewExecutable $view) {
   /**
    * Return the HTML to reference an SVG.
    */
-  public static function getSvgEmbed($hash) {
-    return sprintf('<svg aria-hidden="true" focusable="false"><use xlink:href="#%s"></use></svg>', $hash);
+  public static function getSvgEmbed($hash, $dimensions = []) {
+    // Build dimension attributes
+    $dimensionAttrs = '';
+    if (!empty($dimensions['width'])) {
+      $dimensionAttrs .= ' width="' . htmlspecialchars($dimensions['width']) . '"';
```

Comment:
```suggestion
      $dimensionAttrs .= ' width="' . $dimensions['width'] . '"';
```
You could use `check_plain` instead of htmlspecialchars, which is a Drupal implementation. I think I would just check if the value is numeric though like the suggestion above.